install/config.json

```diff
@@ -0,0 +1,3 @@
+{
+	"portal_host": "172.31.4.20"
```

Comment:
ideally there shouldn't be any hardcoded ip in the checked-in repo.

the issue is when the code is cloned and changed for local environment, this file is always a change in the local repo. other option may be specifying config file, K8s configmap, env variable. Any suggestion?

Reply:
It does not take effect until you specify a host in your cluster. config map should not work since control plane is registered at very beginning. We can set up env variables but we have to set it in all the host.

Reply:
please add the tracking issue id here in comment.